examples/app_multi_node/app_lite_work_executor.py

```diff
@@ -0,0 +1,36 @@
+import torch
+
+import lightning as L
+from lightning.app.components import MultiNode
+from lightning.lite import LightningLite
+
+
+class LitePyTorchDistributed(L.LightningWork):
+    @staticmethod
```

Comment:
I understand that we can go for a `staticmethod` here as we are not using instance vars, but is this something we want to send as best practice on how to derive a `Work` class? I don't want to convey the message that this _must_ be a `staticmethod`.

Reply:
This isn't supported yet. The Work isn't pickable.